daemon/src/lib.rs

```diff
@@ -156,6 +156,13 @@ where
             &oracle_addr,
         )));
 
+        let (endpoint_addr, endpoint_context) = Context::new(None);
+
+        let libp2p_collab_settlement_addr =
+            close_position::taker::Actor::new(endpoint_addr.clone(), executor.clone(), n_payouts)
```

Comment:
:question: Why do we use different terminology for the same protocol? Not sure what the situation was like before the PR, but it would be good for our sanity to decide on close or settlement.

Reply:
I think the old discussion is still open: https://github.com/itchysats/itchysats/discussions/741

honestly, I prefer keeping the old name for now- I inherited the patch from Thomas, who started using the other name.

If we decide to change it later on, we will do so in one sweep.